index.css

```diff
@@ -0,0 +1,153 @@
+/* Style Css */
+
+.books-section {
+  padding: 30px;
+  align-items: center;
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  background-color: aquamarine;
+}
+
+.m-hr {
+  width: 30%;
+  border: 1px solid #000;
+}
+
+.m-h {
+  font-size: 20px;
+  font-weight: bold;
+  margin-left: 1.5rem;
+}
+
+.books {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 60%;
+  border: 2px solid #000;
+  background-color: #fff;
+}
+
+.book-wraper {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  /* margin: 10px; */
```

Comment:
- [ ] [OPTIONAL] Please pay attention to `List` section because it's design is not like required wireframe.

![image](https://user-images.githubusercontent.com/63915024/183522083-0d0629f2-1508-4b9b-80c5-d0fb631705a2.png)